src/goSuggest.ts

```diff
@@ -55,7 +55,22 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				// prevent completion when typing in a line comment
 				const commentIndex = lineText.indexOf('//');
 				if (commentIndex >= 0 && position.character > commentIndex) {
-					return resolve([]);
+					if (!lineText.trim().startsWith('//')) {
+						return resolve([]);
+					}
+					// triggering completions in comments on exported members
+					if (position.line + 1 < document.lineCount) {
+						let nextLine = document.lineAt(position.line + 1).text.trim();
+						let memberType = nextLine.match(/(const|func|type|var)\s*(\w+)/);
```

Comment:
We need atleast one space between the keyword and member name, so `\s*` here should be `s+`

Reply:
Added the right regex.